Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -205,6 +205,10 @@ ActsExamples::SeedingAlgorithm::SeedingAlgorithm(
         });
   }
 
+  m_cfg.seedFinderConfig.experimentCuts.connect(
+      [](const void*, const float& /*bottomRadius*/,
```

Comment:
what is the `void*` for?

Reply:
I think this is in the definition of the connect function

```
  /// Connect anything that is assignable to the function pointer
  /// @param callable The runtime value of the callable
  /// @note The function signature requires the first argument of the callable is `const void*`.
  ///       i.e. if the signature of the delegate is `void(int)`, the
  ///       callable's signature has to be `void(const void*, int)`.
  void connect(function_type callable) {
    if constexpr (kOwnership == DelegateType::NonOwning) {
      m_payload.payload = nullptr;
    }
    m_function = callable;
  }
```

Reply:
The `void*` is from the fact that there's a payload object that the delegate will pass in. This is either treated as the instance of an object that a member function is called on, or in this case it could be an arbitrary pointer.

This is only visible, because you're using a lambda here. If you switch to a simple free function, it won't need this pointer I don't think.

Reply:
thank you for the clarification @paulgessinger